docs/rfc/rfc-8.adoc

```diff
@@ -0,0 +1,410 @@
+|                            |     |     |                                                                     |
+|  +----------------------+  |     |     |  +----------------------+  +-------------------+  +-------------+   |
+|  | Wormhole TokenBridge |--|-----|-----|--| Wormhole TokenBridge |--| L2WormholeGateway |--| PolygonTBTC |   |
+|  +----------------------+  |     |     |  +----------------------+  +-------------------+  +-------------+   |
+|                            |     |     |                                                                     |
++----------------------------+     |     +---------------------------------------------------------------------+
+                                   |
+                                   |     +---------------------------------------------------------------------+
+                                   |     |                                Optimism                             |
+                                   |     |                                                                     |
+                                   |     |  +----------------------+  +-------------------+  +--------------+  |
+                                   +-----|--| Wormhole TokenBridge |--| L2WormholeGateway |--| OptimismTBTC |  |
+                                         |  +----------------------+  +-------------------+  +--------------+  |
+                                         |                                                                     |
+                                         +---------------------------------------------------------------------+
+```
+
+The initial implementation uses the Wormhole bridge to bring tBTC to Arbitrum, Optimism, and Polygon.
+
+Smart contracts involved:
+
+* Wormhole `TokenBridge` on Ethereum is the contract deployed at
+  link:https://etherscan.io/address/0x3ee18b2214aff97000d974cf647e7c347e8fa585[`0x3ee18b2214aff97000d974cf647e7c347e8fa585`].
+* Wormhole `TokenBridge` on Arbitrum is the contract deployed at
+  link:https://arbiscan.io/address/0x0b2402144bb366a632d14b83f244d2e0e21bd39c[`0x0b2402144Bb366A632D14B83F244D2e0e21bD39c`].
+* Wormhole `TokenBridge` on Polygon is the contract deployed at
+  link:https://polygonscan.com/address/0x5a58505a96D1dbf8dF91cB21B54419FC36e93fdE[`0x5a58505a96D1dbf8dF91cB21B54419FC36e93fdE`].
+* Wormhole `TokenBridge` on Optimism is the contract deployed at
+  link:https://optimistic.etherscan.io/address/0x1d68124e65fafc907325e3edbf8c4d84499daa8b[`0x1d68124e65fafc907325e3edbf8c4d84499daa8b`].
+* `L2WormholeGateway` on each chain is a smart contract wrapping and unwrapping
+  Wormhole-specific tBTC representation into the canonical tBTC token on the
+  given chain. This contract needs to be implemented and deployed behind an
+  upgradeable proxy.
+* `ArbitrumTBTC`, `PolygonTBTC`, and `OptimismTBTC` are token contracts with
```

Comment:
what are the limits on the minting authority? Rate limit? Absolute amount? Other? 🤔

Reply:
Both rate limit and absolute are useful, though if we do rate limiting it should be using something mega simple to minimize code. Just an absolute would be enough for me though

Reply:
I was thinking about absolute limit being a governable value in the contract having an authority to mint the token. For Arbitrum, it would be `L2WormholeGateway`, and `L2ArbitrumGateway`. The gateway is upgradeable so we can always add rate limits in the future, if needed.

Reply:
Do we want to include these ideas in this RFC, or flesh them out in a follow-up?